lua/lz/n/loader.lua

```diff
@@ -103,8 +103,10 @@ end
 ---@param plugins string | lz.n.Plugin | string[] | lz.n.Plugin[]
 function M.load(plugins)
     plugins = (type(plugins) == "string" or plugins.name) and { plugins } or plugins
-    ---@param plugin string|lz.n.Plugin
-    vim.iter(plugins):each(function(plugin)
+    ---@cast plugins (string|lz.n.Plugin)[]
+    for _, plugin in pairs(plugins) do
```

Comment:
Does it work if you change it to:

```suggestion
    vim.iter(plugins):each(function(_, plugin)
```

?

Reply:
I will try that and see if that fixes it. And I can give it a shot on the test but im not sure. I dont know why its happening.

Reply:
I might not have a regression test till after that anyway XD

Reply:
That makes it worse actually. It might help me figure out whats going on though.